platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -404,8 +404,8 @@ private void checkInitJudges(@NonNull final EventImpl event) {
         initJudges.judgeFound(event);
         logger.info(
                 STARTUP.getMarker(),
-                "Found init judge {}, num remaining: {}",
-                event::toShortString,
+                "Found init judge %s, num remaining: {}"
+                        .formatted(event.getBaseEvent().getDescriptor()),
```

Comment:
since base event no longer exists, maybe the method should be renamed as well

Reply:
I agree, but I think it's best to save for a later PR. There are hundreds of usages, so the rename would blow this PR up